s3/src/main/scala/akka/stream/alpakka/s3/javadsl/S3.scala

```diff
@@ -820,6 +862,47 @@ object S3 {
   def deleteBucket(bucketName: String, materializer: Materializer): CompletionStage[Done] =
     deleteBucket(bucketName, materializer, Attributes(), S3Headers.empty)
 
+  /**
+   * Delete bucket with a given name
+   *
+   * @see https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteBucket.html
+   * @param bucketName   bucket name
+   * @param system the actor system which provides the materializer to run with
+   * @param attributes attributes to run request with
+   * @return [[java.util.concurrent.CompletionStage CompletionStage]] of type [[Done]] as API doesn't return any additional information
+   */
+  def deleteBucket(bucketName: String,
```

Comment:
Should we deprecate the other overloads that take a `Materializer`?

Reply:
Added deprecations.